beacon-chain/sync/data_columns_reconstruct.go

```diff
@@ -57,21 +57,21 @@ func recoverBlobs(
 			cell := column[blobIndex]
 
 			// Transform the cell as a cKzg cell.
-			var ckzgCell cKzg4844.Cell
-			for i := 0; i < cKzg4844.FieldElementsPerCell; i++ {
+			var ckzgCell kzg.Cell
+			for i := 0; i < kzg.FieldElementsPerCell; i++ {
 				copy(ckzgCell[i][:], cell[32*i:32*(i+1)])
 			}
 
 			cKzgCells = append(cKzgCells, ckzgCell)
 		}
 
 		// Recover the blob.
-		recoveredCells, err := cKzg4844.RecoverAllCells(cellsId, cKzgCells)
+		recoveredCells, err := kzg.RecoverAllCells(cellsId, cKzgCells)
 		if err != nil {
 			return nil, errors.Wrapf(err, "recover all cells for blob %d", blobIndex)
 		}
 
-		recoveredBlob, err := cKzg4844.CellsToBlob(recoveredCells)
+		recoveredBlob, err := kzg.CellsToBlob(recoveredCells)
```

Comment:
The usecase for calling these two methods is to eventually compute the CellsAndKZGProofs -- I would recommend adding a `RecoverCellsAndComputeKZGProofs` method which does this, so that when libraries are updated there is not much that needs to be modified.

Pseudocode to clarify

```go
func RecoverCellsAndKZGProofs(cells []Cell) ([]Cells, []Proofs, error) {
  recoveredCells, err := ckzg.RecoverAllCells(cellsIds, cells) // This is no longer called directly in beacon-chain
  recoveredBlob, err := ckzg.CellsToBlob(recoveredCells) // This is no longer called directly in beacon-chain
  cells, proofs, err := ckzg.ComputeCellsAndProofs(recoveredBlob)
   return cells, proofs, err
}
```

Reply:
This is addressed in https://github.com/prysmaticlabs/prysm/pull/14160